testing/matrix-sdk-test/src/test_json/keys_query_sets.rs

```diff
@@ -946,6 +994,10 @@ impl PreviouslyVerifiedTestData {
             .expect("Can't parse the `/keys/upload` response")
     }
 
+    pub fn carol_unsigned_device_id() -> &'static DeviceId {
+        device_id!("BAZAPVEHGA")
+    }
```

Comment:
again, could do with some documentation here. It wasn't clear to me that this was related to `device_1_keys_payload_carol`.

Reply:
Added some more comments